sdk/core/core-amqp/src/index.ts

```diff
@@ -17,8 +17,8 @@ export { IotHubConnectionConfig } from "./connectionConfig/iothubConnectionConfi
 
 export { CbsClient, CbsResponse } from "./cbs";
 export { Constants } from "./util/constants";
-export { MessageHeader } from "./messageHeader";
-export { MessageProperties } from "./messageProperties";
+export { AmqpMessageHeader } from "./messageHeader";
+export { AmqpMessageProperties } from "./messageProperties";
```

Comment:
Though this changes the API Surface.. event-hubs is not affected since it doesn't use these interfaces.

Only Service Bus V7 is affected